README.md

```diff
@@ -4,34 +4,41 @@
   - Ability to pause/resume the bot using `alt+p`
   - Ability to not re-queue the bot for a new game using `alt+n`
 - Does not surrender games early, by default, allowing it to play out most games, which at the ELO your bot will end up at means there's a decent chance you end up in top 4
   - If you're using this for event pass grinding, it will increase your odds of being top 4 which increases the points earned per minute played (faster pass progression)
 - Draft stage pathing (does not just walk to one point, will walk counter-clockwise to try to ensure it picks up a champ)
-- Basic gold logic to only click based on the gold available
-  - If >= 4, buy xp
-  - If >= 1, attempt 3 champ purchases
-    - After that, if >= 2, re-roll
-- Compiles to release executables so you do not need Python / pip installed in order to use it
-- Can load settings from a config file (if you want to customize how it starts up)
-- Automatically detects your League install location from the Windows registry, allowing non-default installs to still benefit from this script
+- Two gold-based decision-making systems, a default one (requires **no** dependency) and a higher-effort one (requires additional dependency).
+- Automatically detects your League install location from the Windows registry.
+  - This means the bot is able to freely start and restart your league client, should any issues arise, no matter where you installed it.
 
 How stable is this you might ask?
 I recorded this screenshot after a couple days if it running straight, and the battlepass XP matches 😜
 
 ![image](https://user-images.githubusercontent.com/7606153/208268290-8956bfb0-62d4-4d2f-9dd9-0c17c4c1a20e.png)
 
-
-
 # Usage / Settings
 
 * Adding the (`-f` or `--ffearly`) argument will make it forfeit at the first opportunity
 * Adding the (`-v` or `--verbose`) argument will enable more verbose debug logging
   * This toggles whether the verbose logging should log to console / window. Verbose logging will always log to the log file.
 * You can use the config file below to "save" these settings
 
+## Optional: Install Tesseract-OCR & Enable economic decision-making
+
+By default, or if we could not find tesseract on your system, the bot will use a straight-forward logic of constantly leveling, rolling and buying units.
+To change this, we support economic decision-making backed by reading the accurate value of your gold from your screen.
+However, this **requires** Tesseract-OCR to be installed, which you can download a pre-built Windows installer for [here](https://github.com/UB-Mannheim/tesseract/wiki).
+For this bots purpose, you can deselect everything in the components to install to only install the bare minimum.
+
+After you have downloaded and installed Tesseract-OCR, set `mode` unter `economy` to `ocr_standard` in the `config.yaml`.

```

Comment:
```suggestion
After you have downloaded and installed Tesseract-OCR, set `mode` under `economy` to `ocr_standard` in the `config.yaml`.

```

Reply:
Oops, the German brain got through, thanks